helpers/seed.go

```diff
@@ -0,0 +1,114 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/gofrs/uuid"
+	models "github.com/layer5io/meshery/models"
+	"github.com/sirupsen/logrus"
+)
+
+type contentandName struct {
```

Comment:
Is this new struct needed? Can we use the ones for patterns and filters already defined in `models` package? Helps us keep uniformity and maintainability.
You can also then assign UUIDs while reading seed content from the filesystem.

Reply:
this is a redundant struct, removing it anyways